tests/tcp.rs

```diff
@@ -0,0 +1,87 @@
+#[macro_use]
+extern crate unwrap;
+extern crate socket_collection;
+#[macro_use]
+extern crate net_literals;
+extern crate mio;
+extern crate rand;
+
+use mio::net::TcpListener;
+use mio::{Events, Poll, PollOpt, Ready, Token};
+use rand::RngCore;
+use socket_collection::{SocketError, TcpSock};
+
+#[test]
+fn data_transfer_up_to_2_mb() {
+    const SOCKET1_TOKEN: Token = Token(0);
+    const SOCKET2_TOKEN: Token = Token(1);
+    const LISTENER_TOKEN: Token = Token(2);
+    let evloop = unwrap!(Poll::new());
+
+    let listener = unwrap!(TcpListener::bind(&addr!("127.0.0.1:0")));
+    let listener_addr = unwrap!(listener.local_addr());
+
+    let mut sock1 = unwrap!(TcpSock::connect(&listener_addr));
+    let mut sock2 = None;
+
+    unwrap!(evloop.register(&sock1, SOCKET1_TOKEN, Ready::writable(), PollOpt::edge(),));
```

Comment:
we call it `el` in crust and p2p .. let's keep that convention and call event-loops `el` so it's easier to recognise at the first glance by being uniform everywhere.

Reply:
Personally, I'd prefere crust and p2p transitioning to `evloop` :D `el` is just not that descriptive.